packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -468,12 +475,28 @@ export default class NewVm extends BaseComponent {
           _vif.allowedIpv6Addresses.push(ip)
         }
       })
+
+      if (index + 1 <= templateVifs.length) {
```

Comment:
Wouldn't it be more simple like this?
```suggestion
      if (index < templateVifs.length) {
```